jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImplTest.java

```diff
@@ -167,4 +179,117 @@ public void testAbortCommand() {
         verify(presenter).bulkAbort(any());
         verify(table).deselectAllItems();
     }
+
+    @Test
+    public void addDomainSpecifColumnsTest() {
+        final ListTable<ProcessInstanceSummary> currentListGrid = spy(new ListTable<>(new GridGlobalPreferences()));
+        final Set<String> domainColumns = new HashSet<String>(Arrays.asList("var1", "var2", "var3"));
+        getView().addDomainSpecifColumns(currentListGrid,
+                                         domainColumns);
+
+        final ArgumentCaptor<List> argument = ArgumentCaptor.forClass(List.class);
+        verify(currentListGrid).addColumns(argument.capture());
+
+        final List<ColumnMeta> columns = argument.getValue();
+
+        assertThat(columns.size()).isEqualTo(3);
+        assertThat(columns.stream().map(column -> column.getCaption())
+                           .collect(Collectors.toList())).containsAll(domainColumns);
+    }
+
+    @Test
+    public void testRenameProcessVariableForInitColumns(){
+        ProcessInstanceListViewImpl processInstanceListView = view;
```

Comment:
You could just use the _view_ variable in here right? This is done in quite a few methods below as well.